pkg/eks/client.go

```diff
@@ -169,6 +170,15 @@ func WaitForNodes(ctx context.Context, clientSet kubernetes.Interface, ng KubeNo
 				logger.Debug("the watcher channel was closed... stop processing events from it")
 				return fmt.Errorf("the watcher channel for the nodes was closed by Kubernetes due to an unknown error")
 			}
+			if event.Type == watch.Error {
```

Comment:
Yep I tried this. Sadly since we debug log the event before this in every case it actually had a nil event so this didn't do anything. But let's hope that there will be a time when this changes. 😊

Reply:
Nope, it wasn't. I think there is a log about that at some point in one of the runs if you can see that. ¯\_(ツ)_/¯ 

But that doesn't matter, because eventually it might and THEN we will have what went wrong with this. :)